### PR TITLE
feat: instances/search and instances/list

### PR DESCRIPTION
**Pull request by gulkaktus**
Add `models/instances/search` and `models/instances/list` endpoints to `stable`.

### CODE REVIEW

File: packages/stable/src/.cognite-openapi-snapshot.json [hunk 1]
```diff
@@ -57975,18 +57975,6 @@
                       }
                     ]
                   }
-                },
```
Comment (gulkaktus) [inline packages/stable/src/.cognite-openapi-snapshot.json]: Should not remove this. Was only done to make type generation script run

File: packages/stable/src/exports.gen.ts [hunk 2]
```diff
@@ -117,6 +117,129 @@ export {
   Label,
   LabelList
 } from './api/documents/types.gen';
+export {
```
Comment (andersfylling) [inline packages/stable/src/exports.gen.ts]: This file is owned by the codegen package. So whenever someone runs codegen, these imports will be lost.

Simply moving these out of the exports.gen.ts file will resolve the potential future issue.

**Review by andersfylling — CHANGES_REQUESTED** [review]

File: packages/stable/src/exports.gen.ts (hunk 2 above)
Comment (gulkaktus) [reply packages/stable/src/exports.gen.ts]: Thanks for pointing that out! Moved the types to the types.ts file instead.

**Review by andersfylling — APPROVED** [review]
lgtm.

If you make any big chances, besides fixing the test cases, then please request another review.

**Pull request merged**